examples/phoenix_app/lib/phoenix_app_web/controllers/oauth_controller.ex

```diff
@@ -1,7 +1,7 @@
 defmodule PhoenixAppWeb.OauthController do
   use PhoenixAppWeb, :controller
 
-  def access_token(conn = %Plug.Conn{}, _params) do
+  def access_token(%Plug.Conn{} = conn, _params) do
```

Comment:
Changing the pattern matching convention seems unnecessary. Can we disable the credo check and have a separate conversation about the convention? I prefer variable on the left style.

Reply:
I've always used this form, but I'm not really interested in the specific style, I just harmonized it and I think the good point is to have consistency, which is what credo wants with that check.

That said I'll revert this and move it to a different PR.